modules/apps/translation/translation-test/src/testFunctional/tests/Translations.testcase

```diff
@@ -0,0 +1,83 @@
+@component-name = "portal-collaboration"
+definition {
+
+	property osgi.module.configuration.file.names = "com.liferay.journal.web.internal.configuration.FFImportExportTranslationConfiguration.config";
+	property osgi.module.configurations = "enabled=B&quot;true&quot;";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.main.component.name = "Translations Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			User.logoutPG();
+		}
+	}
+
+	@description = "This ensures that a translated zip file with all available language can be exported."
+	@priority = "5"
+	test ExportWCWithAllLanguages {
+		JSONWebcontent.addWebContent(
+			content = "Web Content Content",
+			groupName = "Guest",
+			title = "Web Content Title");
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
+
+		Translations.gotoExportTranslation(webContentTitle = "Web Content Title");
+
+		Translations.exportTranslation(
+			defaultLanguage = "English (United States)",
+			languageList = "Arabic (Saudi Arabia),Catalan (Spain),Chinese (China),Dutch (Netherlands),Finnish (Finland),French (France),German (Germany),Hungarian (Hungary),Japanese (Japan),Portuguese (Brazil),Spanish (Spain),Swedish (Sweden)");
+
+		var translationZipFileName = TestCase.getDownloadedTempFileName(fileNamePattern = "Web Content Title*.zip");
+
+		echo("## translationZipFileName: ${translationZipFileName}");
+
+		if (contains("${translationZipFileName}", "en_US")) {
+			echo("This is a exported translation file.");
+		}
+		else {
+			fail("FAIL!");
+		}
+	}
+
+	@description = "This ensures that a translated zip file with an available language can be exported."
+	@priority = "5"
+	test ExportWebContent {
+		JSONWebcontent.addWebContent(
+			content = "Web Content Content",
+			groupName = "Guest",
+			title = "Web Content Title");
+
+		WebContentNavigator.openWebContentAdmin(siteURLKey = "guest");
+
+		Translations.gotoExportTranslation(webContentTitle = "Web Content Title");
+
+		Translations.exportTranslation(
+			defaultLanguage = "English (United States)",
+			languageList = "Spanish (Spain)");
+
+		var translationZipFileName = TestCase.getDownloadedTempFileName(fileNamePattern = "Web Content Title*.zip");
+
+		echo("## translationZipFileName: ${translationZipFileName}");
+
+		if (contains("${translationZipFileName}", "en_US")) {
+			echo("This is a exported translation file.");
+		}
+		else {
+			fail("FAIL!");
+		}
```

Comment:
Can we make this into a reusable macro?

Reply:
See https://github.com/austinchiang/liferay-portal/pull/831